test/numerical/TestConv.cpp

```diff
@@ -61,21 +58,50 @@ int main(int argc, char *argv[]) {
       argc, argv, "TestConv\n", nullptr, "TEST_ARGS");
   std::cout << "Target options: \""
             << getCompilerOption(OptionKind::TargetAccel) << "\"\n";
-
-  // Had to explicitly iterate over dynamic as otherwise the random algorithm
+  // Get configurations from an environment variable
+  std::map<std::string, std::string> opts =
+      ModelLibBuilder::getTestConfigFromEnv("TEST_CONFIG");
+  // Set configuration for test
+  int dimType, maxDilation;
+  std::string paddingType;
+  if (opts["-dim"] == "static") {
+    std::cout << "Dimension type from env: \"" << opts["-dim"] << "\"\n";
+    dimType = 1; // only static
+  } else {
+    dimType = 2; // static and dynamic (default)
+  }
+  if (opts["-dilation"] == "1") {
+    std::cout << "Dilation from env: \"" << opts["-dilation"] << "\"\n";
+    maxDilation = 2; // dilation = 1
```

Comment:
It might be easier to state in the comment that `maxDilation` in an exclusive upper bound.

Reply:
Done